Player/src/supplementary/experiments/scripts/FindBestBaseAgentScriptsGen.java

```diff
@@ -178,7 +178,7 @@ private FindBestBaseAgentScriptsGen()
 							if (supportStochasticGames && !mctsSupportsStochastic[4][i5])
 								continue;
 							
-							combos.add(TIntArrayList.wrap(i1, i2, i3, i4, i5));
+							combos.add(TIntArrayList.wrap(new int[]{i1, i2, i3, i4, i5}));
```

Comment:
I couldn't find which version this was using, and since the change was trivial decided to use this instead